pkg/kubectl/categories/categories.go

```diff
@@ -62,14 +70,15 @@ func (e discoveryCategoryExpander) Expand(category string) ([]schema.GroupResour
 		if err != nil {
 			return e.fallbackExpander.Expand(category)
 		}
-
+		// Traverse all Category, creating GroupResource object by Group and Resource name, and mapped by category sting as key.
```

Comment:
try to be more abstract and short, maybe something like: collect GroupVersions by categories